usagov/usagov-2021
# USAGOV-1629-Benefits-Category-Language-Filter

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1629

## Description

The `Benefit Search Category Reference` View has been updated so each benefit category will display its language.
The `benefitsCategoryLanguageFilter` script converts that displayed language to a data attribute, and then filters the items by the currently selected language.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Start creating a new `Basic Page`
- Locate the `Benefit Category Search` field
- [ ] Confirm that only the english benefits categories are displayed
- Change the Language field to Español
- [ ] Confirm that only the Spanish benefits categories are displayed

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
